src/frontend/qt_sdl/EmuInstance.cpp

```diff
@@ -93,25 +93,25 @@ EmuInstance::EmuInstance(int inst) : deleting(false),
     if (val == 0.0)
     {
         Platform::Log(Platform::LogLevel::Error, "Target FPS in config invalid\n");
-        targetFPS = 1.0 / 60.0;
```

Comment:
if this variable contains the inverse of the framerate it should be called targetFrametime

Reply:
ah no it is now right, it is FPS now and used to be frametime, everything's alright then